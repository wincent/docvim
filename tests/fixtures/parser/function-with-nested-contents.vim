fu Foo()
  " Contents go here.
  let g:foo=1
endf
