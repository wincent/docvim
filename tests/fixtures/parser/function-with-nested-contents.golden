Project
  [ Project
      [ Unit
          [ FunctionDeclaration
              { functionBang = False
              , functionName = "Foo"
              , functionArguments = ArgumentList []
              , functionAttributes = []
              , functionBody =
                  [ LetStatement { letLexpr = "g:foo" , letValue = "1" } ]
              }
          ]
      ]
  ]
